docs/user-guide.md

```diff
@@ -0,0 +1,223 @@
+  /dev/sda6  filesystem  iso9660  agentdata   18G   -
+  ```
+* The two additional partitions:
+  * `agentboot`: Agent-based installer ISO:
+    * Allows a first boot.
+    * Used as a recovery / re-install partition (with an added grub manu entry).
+  * `agentdata`: OCP release images payload.
+* Note that sizes may change, depending on the configured `diskSizeGB` and the selected OpenShift version configured in `appliance-config.yaml`
+* It’s important to note that the sizes may vary based on the `diskSizeGB` configuration and the OpenShift version selected in the `appliance-config.yaml` file (detailed later on).
+
+### Factory
+* This is where the disk image is written to the disk using tools such as `dd`.
+* As mentioned above, the created image is generic. Thus, the same image can be used on multiple servers, assuming they have the same disk size.
+
+### User site
+* This is where the user boots the machine and mounts the configuration ISO (cluster configuration).
+* The OpenShift installation will run until completion.
+
+:warning: note that the openshift-appliance disk image supports UEFI boot mode only.
+
+
+## Disk Image Build - Lab 
+
+### Set Environment
+
+**:warning: use absolute paths.**
+  ```shell
+  export APPLIANCE_IMAGE="quay.io/edge-infrastructure/openshift-appliance"
+  export APPLIANCE_ASSETS="/home/test/appliance_assets"
+  ```
+
+### Get `openshift-appliance` container image:
+  ```shell
+  podman pull $APPLIANCE_IMAGE
+  ```
+
+### Generate a template of the appliance config
+A configuration file named `appliance-config.yaml` is required for running `openshift-appliance`.
+  ```shell
+  podman run --rm -it -v $APPLIANCE_ASSETS:/assets:Z $APPLIANCE_IMAGE generate-config
+  ```
+Result:
+```shell
+INFO Generated config file in assets directory: appliance-config.yaml
+```
+  
+
+### Set `appliance-config`
+* Initially, the template will include comments about each option and will look as follows:
+
+```yaml
+#
+# Note: This is a sample ApplianceConfig file showing
+# which fields are available to aid you in creating your
+# own appliance-config.yaml file.
+#
+apiVersion: v1beta1
+kind: ApplianceConfig
+ocpRelease:
+  # OCP release version in major.minor or major.minor.patch format
+  # (in case of major.minor - latest patch version will be used)
+  version: ocp-release-version
+  # OCP release update channel: stable|fast|eus|candidate
+  # Default: stable
+  # [Optional]
+  channel: ocp-release-channel
+  # OCP release CPU architecture: x86_64|aarch64|ppc64le
+  # Default: x86_64
+  # [Optional]
+  cpuArchitecture: cpu-architecture
+# Virtual size of the appliance disk image (at least 150GiB)
+diskSizeGB: disk-size
+# PullSecret required for mirroring the OCP release payload
+pullSecret: pull-secret
+# Public SSH key for accessing the appliance
+# [Optional]
+sshKey: ssh-key
+# Password of user 'core' for connecting from console
+# [Optional]
+userCorePass: user-core-pass
+```
+* Modify it based on your needs. Note that:
```

Comment:
Modify the template based on your needs.

Reply:
It is pretty clear what we are referring to here, as it comes right after and is nested under the same section.

Reply:
Well. if you spend a few mins reading the yaml example, then you need to scroll up to see what it is referring to.